apis/io-engine/protobuf/v1/pool.proto

```diff
@@ -61,15 +82,46 @@ enum PoolState {
 
 // Storage pool properties
 message Pool {
-  string uuid = 1;            // uuid of the pool
-  string name = 2;            // name of the pool
-  repeated string disks = 3;  // absolute disk paths claimed by the pool
-  PoolState state = 4;        // current state of the pool
-  uint64 capacity = 5;        // size of the pool in bytes
-  uint64 used = 6;            // used bytes from the pool
-  PoolType pooltype = 7;      // type of the pool
-  uint64 committed = 8;       // committed size of all pool replicas (sum of capacities of all replicas)
-  uint32 cluster_size = 9;    // blobstor cluster size set (in bytes) during pool creation
+  // UUID of the storage pool.
+  string uuid = 1;
+  // Name of the storage pool.
+  string name = 2;
+  // Absolute path(s) of disk devices which are claimed by the pool.
+  repeated string disks = 3;
+  // Current state of the storage pool.
+  PoolState state = 4;
+  // Pool capacity in bytes available for user data.
+  uint64 capacity = 5;
+  // Storage capacity in bytes allocated for storing user data.
+  uint64 used = 6;
+  // Type of the pool.
+  PoolType pooltype = 7;
+  // Committed size of all pool's replicas: a sum of capacities of all replicas.
+  uint64 committed = 8;
+  // Cluster size in bytes. It defines the allocation granularity for replicas
+  // and snapshots. Cluster size can set during pool creation.
+  uint32 cluster_size = 9;
+  // Page size in bytes. This is the write and read granularity.
+  uint32 page_size = 10;
+  // Size of metadata allocation page in bytes.
+  uint32 md_page_size = 11;
+  // Number of metadata pages reserved on the pool for storing replica metadata.
+  // Metadata reservation capacity can be a limiting factor for the number
+  // of replicas and snaphots a pool can accommodate.
+  // Depending on the pool backend, this value can be defined on pool creation,
+  // or can dynamically change.
+  // If the pool backend does not support metadata capacity evaluation, zero is returned.
+  uint64 md_pages = 12;
+  // Number of metadata pages used for storing metadata for the currently
+  // existing replicas.
+  // If the pool backend does not support metadata capacity evaluation, zero is returned.
+  uint64 md_used_pages = 13;
+  // Usable capacity of the pool's underlying storage device(s).
+  // It changes when the device's capacity changes.
+  // This value is equal or large than the current pool capacity. If it is larger,
+  // pool can be grown to fill the entire device's capacity.
+  // If the pool backend does not support disk capacity evaluation, zero is returned.
+  uint64 disk_capacity = 14;
```

Comment:
We could move this to another Message?
If the Message is Some is sent then it's new version, otherwise it's old version?

Reply:
Done